.travis.yml

```diff
@@ -37,8 +37,10 @@ install:
   - if [[ $setup = 'basic' && "$TRAVIS_PHP_VERSION" != "nightly" ]]; then travis_retry composer install --prefer-dist --no-interaction; fi
   - if [[ $setup = 'lowest' && "$TRAVIS_PHP_VERSION" != "nightly" ]]; then travis_retry composer update --prefer-dist --no-interaction --prefer-lowest --prefer-stable; fi
   - if [[ "$TRAVIS_PHP_VERSION" = "nightly" ]]; then travis_retry composer install --prefer-dist --no-interaction --ignore-platform-reqs; fi
+  - if [[ "$TRAVIS_PHP_VERSION" = "7.4" ]]; then travis_retry composer requrie --dev phpstan/phpstan:1.1.2; fi
 
 script:
+  - if [[ "$TRAVIS_PHP_VERSION" = "7.4" ]]; then composer require vendor/bin/phpstan; fi
```

Comment:
`composer require` looks wrong here

Reply:
should be `composer require --dev` ?

Reply:
should be none of it, as this is expected to run phpstan, not to install it

Reply:
I updated it https://github.com/tijsverkoyen/CssToInlineStyles/pull/215/commits/9f076e68e9aa5663a7aa07f5728ade655760204b

Reply:
Also worth pointing out that travis isn't running anymore... Perhaps should switch to GitHub Actions in another PR?